activesupport/test/concern_test.rb

```diff
@@ -17,15 +17,15 @@ def included_ran=(value)
       end
 
       def included_ran
-        @included_ran
+        defined?(@included_ran) ? @included_ran : nil
       end
 
       def prepended_ran=(value)
         @prepended_ran = value
       end
 
       def prepended_ran
-        @prepended_ran
+        defined?(@prepended_ran) ? @prepended_ran : nil
       end
```

Comment:
Could these four methods be replaced with:

```ruby
attr_accessor :included_ran, :prepended_ran
```

?

Reply:
That could be better.

Reply:
Fixed.
Yes, 16 times better by line count, I'm not sure how to quantify the clarity :) Thank you.